tools/deploy.py

```diff
@@ -69,6 +71,8 @@ class _TargetChoices(object):
     configure_logging()
     parser = get_parser()
     parser.add_argument(
+            '-v', '--version', help='Version name for Workbench to deploy')
```

Comment:
If we call deploy.py directly and omit --version this will end up breaking I think (args.version will be None, which will upset gcloud and/or subprocess). Maybe add a default, or build the gcloud argument list dynamically so it doesn't include --version if we didn't pass one in, or make --version a required/positional arg?

Reply:
Made --version required.